Sources/AppcuesKit/Push/UIApplication+AutoConfig.swift

```diff
@@ -35,10 +35,12 @@ extension UIApplication {
         _ application: UIApplication,
         didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data
     ) {
-        AppcuesUNUserNotificationCenterDelegate.shared.didRegister(deviceToken: deviceToken)
```

Comment:
Having the new `PushAutoConfig` is nice because it means there's no out of place reference to `AppcuesUNUserNotificationCenterDelegate` any more.